{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cardExceededLimits"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/processout\/pofailure\/genericcode\/cardexceededlimits"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure\/GenericCode\/cardExceededLimits","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The card limits were reached (ex: amounts, transactions volume) and the customer should contact its bank."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cardExceededLimits"}],"title":"POFailure.GenericCode.cardExceededLimits","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10ProcessOut9POFailureV11GenericCodeO18cardExceededLimitsyA2EmF","modules":[{"name":"ProcessOut"}]},"hierarchy":{"paths":[["doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut","doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure","doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure\/GenericCode"]]},"references":{"doc://com.processout.processout-ios/documentation/ProcessOut":{"role":"collection","title":"ProcessOut","abstract":[],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut","kind":"symbol","type":"topic","url":"\/documentation\/processout"},"doc://com.processout.processout-ios/documentation/ProcessOut/POFailure/GenericCode":{"role":"symbol","title":"POFailure.GenericCode","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"GenericCode"}],"abstract":[],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure\/GenericCode","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"GenericCode"}],"url":"\/documentation\/processout\/pofailure\/genericcode"},"doc://com.processout.processout-ios/documentation/ProcessOut/POFailure":{"role":"symbol","title":"POFailure","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"POFailure"}],"abstract":[{"type":"text","text":"Information about an error that occurred."}],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"POFailure"}],"url":"\/documentation\/processout\/pofailure"},"doc://com.processout.processout-ios/documentation/ProcessOut/POFailure/GenericCode/cardExceededLimits":{"role":"symbol","title":"POFailure.GenericCode.cardExceededLimits","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cardExceededLimits"}],"abstract":[{"type":"text","text":"The card limits were reached (ex: amounts, transactions volume) and the customer should contact its bank."}],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/POFailure\/GenericCode\/cardExceededLimits","kind":"symbol","type":"topic","url":"\/documentation\/processout\/pofailure\/genericcode\/cardexceededlimits"}}}